pkg/apis/operators/v1alpha1/servicebinding_types.go

```diff
@@ -47,6 +47,14 @@ type ServiceBindingSpec struct {
 	BindAsFiles bool `json:"bindAsFiles,omitempty"`
 }
 
+// ServiceBindingMapping defines a mapping from the existing collection of Secret values to a new Secret entry.
```

Comment:
Can we remove `ServiceBinding` prefix from the struct name, it is redundant. Also, can we tailor the description to user facing abstraction, i.e. binding:

```
defines a new binding from set of existing bindings
```